lib/passport-wsfed-saml2/wsfederation.js

```diff
@@ -30,6 +30,11 @@ WsFederation.prototype = {
   extractToken: function(req) {
     var doc = new xmldom.DOMParser().parseFromString(req.body['wresult']);
     var token = doc.getElementsByTagNameNS('http://schemas.xmlsoap.org/ws/2005/02/trust', 'RequestedSecurityToken')[0];
+
+    //if the token was not found, search for the namespace that WIF sets
+    if(!token) {
+      token = doc.getElementsByTagNameNS('http://docs.oasis-open.org/ws-sx/ws-trust/200512', 'RequestedSecurityToken')[0];

```

Comment:
maybe you can set the namespace from options (default: `http://schemas.xmlsoap.org/ws/2005/02/trust`)